keps/sig-node/4680-add-resource-health-to-pod-status/README.md

```diff
@@ -245,7 +290,39 @@ The API will be limited to "prepared" devices and include the claim `name/namesp
 
 Kubelet will react on this field the same way as we propose to do it for the Device Plugin.
 
-Specific implementation details will be added for the beta.
+The new method will be added to [Node service](https://github.com/kubernetes/kubernetes/blob/04bba3c222bb2c5b1b1565713de4bf334ee7fbe4/staging/src/k8s.io/kubelet/pkg/apis/dra/v1alpha4/api.proto#L34)
+alongside `NodePrepareResources` and `NodeUnprepareResources` and a `health` field for `Device` structure:
+
+```
+service Node {
+  ...
+
+	// WatchDevicesStatus returns a stream of List of Devices
+	// Whenever a Device state change or a Device disappears, WatchDevicesStatus
+	// returns the new list.
+  // This method is optional and may not be implemented.
+	rpc WatchDevicesStatus(Empty) returns (stream DevicesStatusResponse) {}
+}
+
+// ListAndWatch returns a stream of List of Devices
+// Whenever a Device state change or a Device disappears, ListAndWatch
+// returns the new list
+message DevicesStatusResponse {
+	repeated Device devices = 1;
+}
+
+message Device {
+  ... existing fields ...
+    // The device itself. Required.
+    string device_name = 3;
+  ... existing fields ...
+
+  // Health of the device, can be Healthy or Unhealthy.
```

Comment:
```suggestion
  // Health of the device, can be Healthy or Unhealthy.
```

In what case, would we get Unknown?

Reply:
we consider adding this for the cases when Device Plugin simply disappeared, but we keep devices around for the graceful period. It is not implemented yet though